internal/test/suite/suite.go

```diff
@@ -0,0 +1,71 @@
+// Package suite is a simplified version of testify's suite package which has unnecessary dependencies.
+// Please remove this package whenever possible.
+package suite
+
+import (
+	"flag"
+	"fmt"
+	"reflect"
+	"runtime/debug"
+	"strings"
+	"testing"
+	"time"
```

Comment:
Build is failing here; https://ci.docker.com/public/job/moby/job/PR-39799/7/execution/node/156/log/

```
23:20:11  internal/test/suite/suite.go:1::warning: file is not goimported (goimports)
23:20:11  internal/test/suite/suite.go:7:2:warning: "fmt" imported but not used (unconvert)
23:20:11  internal/test/suite/suite.go:12:2:warning: "time" imported but not used (gosimple)
```